tests/phpunit/tests/file.php

```diff
@@ -262,4 +262,40 @@ public function filter_trust_plus85Tq_key( $keys ) {
 
 		return $keys;
 	}
+
+	/**
+	 * @ticket 49412
+	 */
+	function test_wp_filesize_with_nonexistent_file() {
+		$file = 'nonexistent/file.jpg';
+		$this->assertEquals( 0, wp_filesize( $file ) );
+	}
+
+	/**
+	 * @ticket 49412
+	 */
+	function test_wp_filesize() {
+		$file = DIR_TESTDATA . '/images/test-image-upside-down.jpg';
+
+		$this->assertEquals( filesize( $file ), wp_filesize( $file ) );
+
+		$filter = function() {
+			return 999;
+		};
+
+		add_filter( 'wp_filesize', $filter );
+
+		$this->assertEquals( 999, wp_filesize( $file ) );
+
+		$pre_filter = function() {
+			return 111;
+		};
+
+		add_filter( 'pre_wp_filesize', $pre_filter );
+
+		$this->assertEquals( 111, wp_filesize( $file ) );
+
+		remove_filter( 'wp_filesize', $filter );
+		remove_filter( 'pre_wp-filesize', $pre_filter );
```

Comment:
```suggestion
```

Filters are reset during tear down so these are redundant. 🔢 Applies elsewhere in PR too.

Reply:
I don't see the issue here and other tests do this.

Reply:
You're right, it's harmless but not needed. Removing it saves needless busy work, both in terms of computer cycles and commits cleaning it up such as https://core.trac.wordpress.org/changeset/50463